CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Suggest installing monolog to send log messages directly to sentry (#908)
```

Comment:
Can you please make the first letter of the `monolog` and `sentry` words uppercase?